apps/storybook/styles/globals.css

```diff
@@ -0,0 +1,27 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
```

Comment:
another rebase error, i think, all of the code below the 3 lines for tw should be gone.